README.md

```diff
@@ -1,2 +1,462 @@
-# certificate
+# Certificate System Role
+
 Role for managing TLS/SSL certificate issuance and renewal
+
+Linux system role to issue and renew SSL certificates.
+
+Basic usage:
+
+```yaml
+
+---
+- hosts: webserver
+
+  vars:
+    certificate_requests:
+      - name: mycert
+        dns: www.example.com
+        ca: self-sign
+
+  roles:
+    - certificate
+```
+
+On a RPM-based system this will place the certificate in `/etc/pki/tls/certs/mycert.crt`
+and the key in `/etc/pki/tls/private/mycert.key`.
+
+
+## Variables
+
+| Parameter               | Description                                                                                                    | Type | Required | Default           |
+|-------------------------|----------------------------------------------------------------------------------------------------------------|:----:|:--------:|-------------------|
+| certificate_wait        | If the task should wait for the certificate to be issued.                                                      | bool | no       | yes               |
```

Comment:
wait for all certificates in the certificate_requests list?

Reply:
https://github.com/linux-system-roles/certificate/issues/31

Reply:
Yes. It waits for all. Thanks for opening the issue.